german/the-black-hack.md

```diff
@@ -316,20 +317,20 @@ Beim Würfeln, um festzustellen ob sich Eigenschaften erhöhen, würfeln Klerike
 
 ### Göttliche Anrufungen
 
-Ab der zweiten Stufe können Kleriker täglich eine gewisse Anzahl von Göttlichen Anrufungen durchführen, siehe den Abschnitt zu Zaubersprüchen.
+Ab der zweiten Stufe können Kleriker täglich eine gewisse Anzahl von göttlichen Anrufungen durchführen, siehe den Abschnitt zu Zaubersprüchen.
```

Comment:
Ja, da hast du recht. Fällt in den allgemeinen "Regelbegriffe groß oder klein?"-Fall.